### DISCUSSION TITLE
I cannot figure out what installer type does this app use

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Scoop#5774] [Help and Questions] [answered]
I'm trying to create the app manifest for Bisq and nothing works as the installer type is not something common as Inno, MSI, .... and the innounp, msiextract, lessmsi, dark, .... all fail.  Even Universal Extractor 2.0.0 RC3 cannot determine the installer used.

BISQ - https://bisq.network

https://github.com/bisq-network/bisq/releases/download/v1.9.14/Bisq-64bit-1.9.14.exe

**Comment by r15ch13** [comment, marked as answer]
It's build with jpackage. See [WindowsPackage.kt](https://github.com/bisq-network/bisq/blob/master/build-logic/packaging/src/main/kotlin/bisq/gradle/packaging/jpackage/package_formats/WindowsPackage.kt)

By using the `-t#` flag for `7z` you get the files `1`, `2.msi` and `3`. `1` and `3` can be ignored.
The `2.msi` can then be extracted with `lessmsi`

```
7z x -t# Bisq-64bit-1.9.14.exe *.msi
lessmsi x 2.msi
```

**Reply by anderlli0053** [reply]
Thank you!
